common/src/main/java/com/revenuecat/purchases/common/utils.kt

```diff
@@ -18,7 +18,7 @@ import java.util.Locale
 const val MICROS_MULTIPLIER = 1_000_000
 
 fun Purchase.toHumanReadableDescription() =
-    "${this.sku} ${this.orderId} ${this.purchaseToken}"
+    "${this.skus.joinToString()} ${this.orderId} ${this.purchaseToken}"
```

Comment:
this will produce 
`sku1, sku2, sku3 order_id purchase_token`, right? 
maybe for readability purposes, we could add the prefix and suffix? i.e.: 
```kotlin
"${this.skus.joinToString(prefix = "[", postfix = "]")} ${this.orderId} ${this.purchaseToken}"
```
or even add some description, i.e.: `skus: [x, y, z], order_id: foo, token: bar`